.github/workflows/main.yml

```
@@ -69,112 +75,173 @@ jobs:
+        run: |
+          mkdir -p ~/.ssh
+          echo "$SSH_PRIVATE_KEY" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan -H ${{ secrets.EC2_STATIC_IP }} >> ~/.ssh/known_hosts
 
-          sudo apt install awscli
+      - name: Rollback Deployment
+        env:
+          EC2_HOST: ${{ secrets.EC2_STATIC_IP }}
+          NAMESPACE: ${{ secrets.NAMESPACE }}
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:12: Double quote to prevent globbing and word splitting

(shellcheck)